app/views/simulation.scala.html

```diff
@@ -47,48 +48,55 @@
     )
 
     <!-- This can be used to insert nlogo code into this page -->
-    <script type="text/nlogo" id="nlogo-code" date-filename="model.nlogo"></script>
+    <script type="text/nlogo" id="nlogo-code" data-filename="model.nlogo"></script>
 
     <script>
       var modelContainer = document.querySelector("#netlogo-model-container");
       var index          = window.location.href.indexOf("?");
       var nlogoScript    = document.querySelector("#nlogo-code");
       var standaloneURL  = "@routes.Local.standalone.absoluteURL";
+      var pageTitle      = function(modelTitle) {
+        if (modelTitle != null && modelTitle != "") {
+          return "NetLogo Web: " + modelTitle;
```

Comment:
The convention is "Document Name - Software Name" (or in this case: "modelTitle - NetLogo Web").